patches/tools-git-move_source_file.py.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/tools/git/move_source_file.py b/tools/git/move_source_file.py
+index ab57ae721179f77bb92b808e65dfe1ced8cf605f..affc0ac8142d6575176675b413d0920a1c2139ec 100755
+--- a/tools/git/move_source_file.py
++++ b/tools/git/move_source_file.py
+@@ -115,6 +115,7 @@ def UpdatePostMove(from_path, to_path):
+   mffr.MultiFileFindReplace(
+       r'(//.*)%s' % re.escape(from_path),
+       r'\1%s' % to_path,
++      ['*.gni', '*.gn'] + # Brave uses full paths (start with //) to add files
```

Comment:
we sometimes do this, but so does upstream so not sure why this is needed?

Reply:
upstream example
```
java_cpp_enum("chrome_android_java_enums_srcjar") {
  sources = [
    "//chrome/browser/android/customtabs/detached_resource_request.h",
    "//chrome/browser/android/explore_sites/explore_sites_bridge.h",
    "//chrome/browser/android/explore_sites/explore_sites_feature.h",
    "//chrome/browser/android/feedback/connectivity_checker.cc",
    "//chrome/browser/android/policy/policy_auditor.cc",
    "//chrome/browser/android/webapk/webapk_installer.h",
    "//chrome/browser/long_screenshots/long_screenshots_tab_service.h",
    "//chrome/browser/notifications/notification_handler.h",
    "//chrome/browser/notifications/notification_platform_bridge_android.cc",
  ]
}
```

Reply:
it's not common so maybe the script doesn't handle them correctly?

Reply:
maybe an upstream commit since they do use this pattern in some places?

Reply:
Well, I could do an upstream CL, but anyway we need to add `sources.gni` here.